ui/src/app/tasks/task-apps/task-apps.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs/Subscription';
+import { ToastyService, ToastyConfig, ToastOptions, ToastData } from 'ng2-toasty';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+import { AppRegistration } from '../../apps/model/app-registration';
+import { Page } from '../../shared/model/page';
+import { AppsService } from '../../apps/apps.service';
```

Comment:
@jvalkeal This is an interesting - one. I don't think that a feature module should import another feature-module's service. I wonder if that means, we should move some of the apps.service methods into the share module.